src/mewbot/behaviour.py

```diff
@@ -75,29 +21,31 @@ def __init__(self) -> None:
         self.conditions = []
         self.actions = []
 
-    def add(self, block: BehaviourComponent) -> None:
-        if not isinstance(block, BehaviourComponent):
+    def add(
```

Comment:
Future Thoughts: One "setup" function which takes in the three Trigger / Condition / Action
Use case: setting up a behaviour on one line in bot init

Reply:
Not needed, configuration should never be in python

Reply:
(more likely is that this will get changed to adding an iterable of that the current type, a la
```python
add(self, *components: Union[ *Interface... ]
``` and then adding all of them by the existing method.